lib/node/init.mjs

```diff
@@ -3,5 +3,8 @@ import Init from "../../dist/node/init.mjs";
 const { main } = Init({ log: "info" });
 
 const root = process.argv[2] || process.cwd();
-const ret = await main(root);
-process.exit(ret ? 0 : 1);
+const p = main(root).then((ret) => {
+  process.exit(ret ? 0 : 1);
+});
+
+Promise.all([p]);
```

Comment:
@lachrist The new and the old code seem to behave the same way, but are they guaranteed to be equivalent?